yetanotherco/aligned_layer
# feat(explorer): mark pending batches in red after 5 minutes

## Description

Marks pending batches in red after 5 minutes

**Testing**
1. Start a local devnet
2. Run the explorer
3. Kill operators so that batches don't reach quorum
4. Send proofs
5. Verify that after 5min it goes red.

## Type of change

## Checklist

  - [Link to the open issue addressing it]()
